packaging/nuget/NServiceBus.Transport.RabbitMQ.nuspec

```diff
@@ -1,25 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
-<package xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
-  <metadata xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
-    <id>NServiceBus.RabbitMQ</id>
-    <title>NServiceBus RabbitMQ</title>
```

Comment:
Decided to go ahead and remove it in anticipation if it going away. Makes more sense to just use package id anyway.